src/Console/Commands/RebuildUriCache.php

```diff
@@ -45,12 +45,14 @@ public function __construct()
      */
     public function handle()
     {
-        $confirm = $this->confirm(
-            'You are about to rebuild your entire URI cache. This may take part of your site down while running. Are you sure you want to continue?'
-        );
-
-        if (! $confirm) {
-            return;
+        if (!$this->option('no-interaction')) {
```

Comment:
Can we use `--force` instead?

Laravel does this for its commands, like [`migrate:fresh`](https://github.com/laravel/framework/blob/10.x/src/Illuminate/Database/Console/Migrations/FreshCommand.php#L111) which need user confirmation before doing anything.

Reply:
Absolutely. Just made the changes and added description to input options.